package.json

```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "@types/marked": "^5.0.1",
     "@vueuse/head": "^1.1.26",
+    "axios": "^1.5.0",
```

Comment:
Is axios required or can be it replaced by a standard `fetch()` call?

Reply:
Based on my past experience in fetching API Data it's way better to use Axios then the fetch, and I am not sure if just fetch can manage to do the job, I will look into it

Reply:
I've looked more into it, and yes, it's possible just using `fetch()`, but it somehow seems that the rate limit gets exceeded way faster, I will look more into it.

Reply:
> .I've looked more into it, and yes, it's possible just using fetch(), ~~but it somehow seems that the rate limit gets exceeded way faster~~, I will look more into it.

It seems that it works just like Axios, so no issues, should I remove Axios and push the simple `fetch()` code and remove Axios?

Reply:
Yes

Reply:
> Yes

All good, I've pushed the code